package.json

```diff
@@ -101,7 +101,7 @@
 		"@wordpress/hooks": "2.8.0",
 		"@wordpress/html-entities": "2.7.0",
 		"@wordpress/i18n": "3.11.0",
-		"@wordpress/icons": "2.0.0",
+		"@wordpress/icons": "2.9.0",
```

Comment:
Updated to `2.9` to use the template-part-sidebar icon, this will mostly likely get renamed to sidebar in the next version.